test/zoonk_web/live/follow_live_test.exs

```diff
@@ -0,0 +1,109 @@
+defmodule ZoonkWeb.FollowLiveTest do
+  use ZoonkWeb.ConnCase, async: true
+
+  import Phoenix.LiveViewTest
+  
+  alias Zoonk.Accounts
+
+  describe "Follow page" do
```

Comment:
No need to over engineer this and test all links. Just test if we're assigning global/brazil links properly.

Plus, you're not using the phoenix_test library for test assertions like we do in other tests.